docker/Dockerfile.base

```diff
@@ -58,18 +58,13 @@ RUN git clone https://github.com/glfw/glfw.git && \
   cd ../../ && \
   rm -rf glfw
 
-# MuJoCo 1.5 (for gym)
-RUN mkdir -p /root/.mujoco && \
-  wget https://www.roboti.us/download/mjpro150_linux.zip -O mujoco.zip && \
-  unzip mujoco.zip -d $HOME/.mujoco && \
-  rm mujoco.zip
-ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/root/.mujoco/mjpro150/bin
-
 # MuJoCo 2.0 (for dm_control)
 RUN mkdir -p /root/.mujoco && \
   wget https://www.roboti.us/download/mujoco200_linux.zip -O mujoco.zip && \
   unzip mujoco.zip -d $HOME/.mujoco && \
-  rm mujoco.zip
+  rm mujoco.zip && \
+  cp -r $HOME/.mujoco/mujoco200_linux $HOME/.mujoco/mujoco200
```

Comment:
if `mujoco200_linux` is the new canonical path used by MuJoCo then let's keep that throughout. Lots of libraries may hardcode into that path looking for MuJoCo.

Reply:
I think dm_control was complaining about not finding mujoco200, and gym wanted mujoco200_linux, so i kept both around for now before i test on a clean install. Also, this PR is still a WIP as the title suggests, submitted it just to see the tests on CI

Reply:
I mixed them up. Mujoco-py is the one that looks for mujoco200 while the canonical name is mujoco200_<linux/macos>. So i ended up creating a symlink mujoco200 to mujoco200_linux